tests/unit_test/surveymonkey/surveymonkey_etl_test.py

```diff
@@ -55,20 +55,188 @@ def test_should_ignore_other_field(self):
         )
         assert result.get("other") is None
 
-    def test_should_extract_question(self):
+    def test_should_extract_question_answer_options_even_answers_field_not_exist(self):
         result = get_bq_json_for_survey_questions_response_json({
             **DEFAULT_SURVEY_QUESTIONS_RESPONSE_JSON,
             "pages": [{
                 "questions": [{
                     "id": "Q_ID",
                     "headings": [{
                         "heading": "Is this the question?"
+                    }],
+                    "family": "matrix",
+                    "subtype": "rating",
+                }]
+            }]
+        })
+        assert result["questions"] == [
+            {
+                "question_id": "Q_ID",
+                "question_title": "Is this the question?",
+                "question_type": "matrix",
+                "question_subtype": "rating",
+                "question_answers": [{
+                    "choices": [{
+                        "id": None,
+                        "text": None,
+                        "type": None,
+                        "weight": None
+                    }],
+                    "other": [{
+                        "id": None,
+                        "text": None,
+                        "type": None,
+                        "weight": None
+                    }],
+                    "rows": [{
+                        "id": None,
+                        "text": None,
+                        "type": None,
+                        "weight": None
+                    }],
+                    "cols": [{
+                        "id": None,
+                        "text": None,
+                        "type": None,
+                        "weight": None
                     }]
                 }]
+            }
+        ]
+
+    def test_should_extract_question_answers(self):
+        result = get_bq_json_for_survey_questions_response_json({
+            **DEFAULT_SURVEY_QUESTIONS_RESPONSE_JSON,
```

Comment:
It might be useful to explicitly set the values here. Otherwise reading this test it is not clear why we should for example expect there to be a choice with the text `This is the choice`.
Alternatively, if the test is rather about to test just that all of the elements are extracted without testing the specifics, then you could consider moving the expected result to a constant close to the response json constant.
You could then add more specific tests for `parse_answer_options_in_question_answer_json`.